lib/rabbit_client/consumer.rb

```diff
@@ -13,10 +13,10 @@ class << self
       def configure(opts)
         @queue_name = opts[:queue]
         configuration = {
-          exchange: opts[:exchange],
-          exchange_type: opts[:exchange_type] || 'fanout',
-          heartbeat: opts[:heartbeat] || 60,
-          arguments: { :'x-dead-letter-exchange' => "#{@queue_name}-retry" }
+        exchange: opts[:exchange],
```

Comment:
Was this a rubocop warning? Anyways, I think we should try to make rubocop/rubymine indent this as it was previously.